rudder-web/src/main/webapp/style/rudder/rudder-menu.css

```diff
@@ -559,7 +559,7 @@ a.sidebar-toggle{
 }
 .content-wrapper td, .content-wrapper th {
   padding: 4px 3px;
-  line-height: normal;
+  line-height: 1.42857143;
```

Comment:
https://stackoverflow.com/questions/19982651/why-does-bootstrap-set-the-line-height-property-to-1-428571429